packages/bundle-source/src/index.js

```diff
@@ -8,6 +8,7 @@ import * as babelParser from '@babel/parser';
 import babelGenerate from '@babel/generator';
 import babelTraverse from '@babel/traverse';
 import { makeArchive } from '@endo/compartment-mapper/archive.js';
+import { makeNodeReadPowers } from '@endo/compartment-mapper/src/node-powers.js';
```

Comment:
Be aware that we're trying to deprecate deep links into the `src` directory of a package.

Reply:
Thanks for catching this. I’d fixed the underlying problem with this morning’s publish and forgot to update this line.